bot/cogs/loop.py

```diff
@@ -0,0 +1,40 @@
+import io
+from datetime import datetime, time
+
+import discord
+import pytz
+from discord.ext import commands, tasks
+
+from bot.bot import WarnetBot
+from bot.config import ADMIN_CHANNEL_ID, CustomRoleConfig, GUILD_ID
+
+
+class Loop(commands.Cog):
+    def __init__(self, bot: WarnetBot) -> None:
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_connect(self) -> None:
+        self._monthly_booster.start()
+
+    @tasks.loop(time=time(hour=18))
```

Comment:
Is there a reason why the task needs to be looped for 18 hours instead of 24?

Reply:
It doesn't loop every 18 hours; rather, it occurs daily at 18:00 UTC time. 
I am still looking for a way to change tzinfo with the Jakarta time zone. If I declare `tzinfo = pytz.timezone('Asia/Jakarta')`, it will trigger a warning as shown in the image below.
![image](https://github.com/warnet-gi/WarnetBot/assets/68109788/bf75a295-ebc6-4f1b-b1c0-02c62761b4ff)

Reply:
This could be an idea to invoke the loop with local time
https://github.com/warnet-gi/WarnetBot/blob/c4f528f3f4d65892adec9337acdf62ff4b1d18da/bot/cogs/general.py#L158-L169

But if it's failed, using UTC is fine